src/Data/Text/Lazy/Read.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE OverloadedStrings, CPP #-}
-{-# LANGUAGE Safe #-}
+{-# LANGUAGE Trustworthy #-}
```

Comment:
I'm not really sure how `Safe`/`Trustworthy` work, but I think this change could harm downstream users who infer safety. Can someone who understands the SafeHaskell system comment?

Reply:
I don't think so, `Safe` and `Trustworthy` are indistinguishable for an external observer.

Reply:
Consumer modules marked `Safe` will break, but they're also protected by version bounds here. Let's leave this one up to PVP, and people can migrate as needed.

Reply:
Let me emphasize: `Safe` and `Trustworthy` are indistiguishable for consumers, there is not breakage. 
(If you are wondering what's the point of Safe Haskell, when anyone can slap `Trustworthy` and launch missiles, I don't have an answer other than Safe Haskell is a hoax)